### PR TITLE
[TDP][Docupdates][1725185]

### PR DESCRIPTION
**Pull request by KirtiPereira**
Teams Developer Portal

### CODE REVIEW

File: msteams-platform/concepts/build-and-test/teams-developer-portal.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+title: Manage your apps with the Developer Portal for Microsoft Teams
+description: Learn manage your apps using the Developer Portal for Microsoft Teams
+keywords: getting started developer portal teams
+localization_priority: Normal
+ms.topic: overview
+---
+
+# Developer Portal Overview
+
+The Developer Portal enables app creators to register apps with the Teams app service and publish to an organization's app catalog, or the Teams store. You can invite colleagues in your organization to collaborate on an app, configure runtime environment, and more.
```
Comment (karthigb) [inline msteams-platform/concepts/build-and-test/teams-developer-portal.md:11]: RE: "_The Developer Portal enables app creators to register apps with the Teams app service **and publish to an organization's app catalog, or the Teams store_**"

We should not mention publish in the value proposition statement for Dev Portal. These actions are less seldom used and not the primary purpose of DP.

How about:

"**Easily register, configure and manage your Teams apps from the Developer Portal.**"

File: msteams-platform/concepts/tdp-configuration.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+* The name of the parameter as it appears in the Teams client and is included in the user request
+* A user-friendly title
+* An optional description
+
+> [!NOTE]
+> To create messaging extension using app studio, see [create messaging extension using app studio](~/resources/create-messaging-extension-using-appstudio.md).
+
+##### Meeting extension
+
+//TODO: Rajesh R.
```
Comment (karthigb) [inline msteams-platform/concepts/tdp-configuration.md:87]: @heath-hamilton @surbhigupta12 I'll ping Rajesh to add his paragraph here

File: msteams-platform/concepts/tdp-configuration.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+#### Languages
+
+Set up or change the languages that your app supports.
+
+#### Single Sign-On
+
+Configure your app to authenticate users with single sign-on (SSO).
+
+#### Domains
+//TODO
```
Comment (karthigb) [inline msteams-platform/concepts/tdp-configuration.md:106]: @heath-hamilton You can base this content on:
https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema#validdomains

File: msteams-platform/concepts/tdp-configuration.md (hunk 2 above)
Comment (karthigb) [reply msteams-platform/concepts/tdp-configuration.md:87]: You can also pull the content from here:
https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/what-are-messaging-extensions

File: msteams-platform/concepts/tdp-tools.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+keywords: tools adaptive card scene studio bot
+localization_priority: Normal
+ms.topic: conceptual
+---
+
+# Tools in Developer Portal
+
+### Bot Management
+
+//Todo
```
Comment (karthigb) [inline msteams-platform/concepts/tdp-tools.md:13]: Karthig to do

File: msteams-platform/concepts/tdp-tools.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+Microsoft has a Scene studio that allows you to build the scenes in Together Mode. It is available on the [Scenes Editor - Teams Developer Portal](https://dev.teams.microsoft.com/scenes).
+
+![Scene studio](~/assets/images/apps-in-meetings/scene-design-studio.png)
+
+For more information, see [Build a scene using Scene studio](../apps-in-teams-meetings/teams-together-mode.md#build-a-scene-using-the-scene-studio).
+
+### Adaptive card editor
+
+//Todo
```
Comment (karthigb) [inline msteams-platform/concepts/tdp-tools.md:25]: karthig to do

File: msteams-platform/concepts/tdp-tools.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+### Microsoft identity platform management
+
+//Todo
```
Comment (karthigb) [inline msteams-platform/concepts/tdp-tools.md:29]: karthig to do

File: msteams-platform/concepts/tdp-tools.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+### Teams store app validation
+
+//Todo
```
Comment (karthigb) [inline msteams-platform/concepts/tdp-tools.md:33]: karthig to do

**Review by karthigb — CHANGES_REQUESTED** [review]

File: msteams-platform/concepts/tdp-configuration.md (hunk 2 above)
Comment (v-vrajaraman) [reply msteams-platform/concepts/tdp-configuration.md:87]: This links is for Messagin Extensions. Can you please provide the correct link for Meeting Extensions?

**Comment by v-vrajaraman** [comment]
@karthigb Please provide some more information on Permission, Languages, and Single Sign On. Also, please provide relevant screen shots for Configure Page.

**Comment by v-vrajaraman** [comment]
@karthigb Please update information for all Todo (Meeting Extension in Configure, and all Tools information).

**Comment by v-vrajaraman** [comment]
@karthigb Please add information on the Advanced section.

**Review by heath-hamilton — COMMENTED** [review]
@karthigb Please see the latest draft content. Thanks in advance!

**Review by surbhigupta12 — APPROVED** [review]
Approved.

**Review by heath-hamilton — APPROVED** [review]
Approved.

**Review by karthigb — APPROVED** [review]

**Pull request merged**